docs/en/base/typography.md

```diff
@@ -5,49 +5,51 @@ table_of_contents: true
 
 ## Typography
 
-All text in Vanilla uses the Ubuntu typeface.
+<hr>
+
+All text in Vanilla uses the Ubuntu typeface, which can be downloaded [here](https://design.ubuntu.com/font/)
```

Comment:
Rephrase this not to use a "here" link. Link text should be descriptive of the page to which it links - here "Ubuntu typeface" would make a good link, for instance.